### ISSUE TITLE
Can this be done more efficiently?

### ISSUE THREAD

**Issue by jmchamber58** [realpython/python-basics-exercises#44]
I notice you doing the same thing repeatedly for nouns, verbs, and adjectives. Would a for loop work here to loop through a range (3) for items in noun, verb, adjective? I can't figure out how to do it though.

**Comment by somacdivad** [comment]
@jmchamber58 Yes, it could be done with less code. I will make a second solution available that does not contain so much repetition.
